ballista/scheduler/scheduler_config_spec.toml

```diff
@@ -24,6 +24,11 @@ conf_file_param = "config_file"
 name = "version"
 doc = "Print version of this executable"
 
+[[param]]
+name = "advertise_host"
```

Comment:
I love the use of the word `advertise` as that is what it is commonly called. I dislike the word `host`, because this is hostname & port. Sadly the term for that appears to be "socket", which I think no one uses outside OS libraries https://stackoverflow.com/questions/34955747/whats-the-combination-of-an-ip-address-and-port-number-called

Reply:
How about `advertise_endpoint` ?